History/js/new.js

```diff
@@ -5,24 +5,6 @@
 /* global historyModal:true */
 
 /**
- * Message handlers for integration with extension framworks.
- */
-var messaging = {
-  
-  /**
-   * Attach the message listeners to the interface between the extension
-   * and the injectable application.
-   */
-  initialize: function() {
-    privlyExtension.initialContent = function(){};
-    privlyExtension.messageSecret = function(){};
-      
-    // Initialize message pathway to the extension.
-    privlyExtension.firePrivlyMessageSecretEvent();
```

Comment:
I believe I removed this. It is better we assume the content script is controlled by the host page and only use the content script <-> privly app message pathway for messages that are not privileged, like content height.